### DISCUSSION TITLE
Error 404 when opening pages

### DISCUSSION THREAD

**Discussion by juliermefelix** [serenity-is/Serenity#7107] [Help]
Script error when opening pages.
Error when running serenity 8+, the pages do not open, error 404 appears, I need help to resolve it.

![image](https://github.com/serenity-is/Serenity/assets/7516443/f4655a38-747c-44fe-bdb1-3d76e6a13236)

![image](https://github.com/serenity-is/Serenity/assets/7516443/0fc8e906-1144-451f-8521-c413e01d2ef0)

**Comment by juliermefelix** [comment]
This happened when updating serene to the newest version.
Is updating the serene version to the current one possible?

**Comment by VictorTomaili** [comment]
You may miss some steps while upgrading like editing bundles.json, npm install etc.
